### DISCUSSION TITLE
Unable to create user.

### DISCUSSION THREAD

**Discussion by mzch** [zitadel/zitadel#4618] [Q&A] [answered]
Hi,

I installed Zitadel on my server and tried to create a new user, but failed.
<img width="1518" alt="ScreenShot" src="https://user-images.githubusercontent.com/621178/198200510-816fb38c-80e4-4277-bb98-126f8ff7c5eb.png">

No browser log nor server log. Could anyone tell me how to fix this issue?

**Comment by hifabienne** [comment, marked as answer]
Hi @mzch 

We currently check if the username is an email address, actualy if it has an @ in between.
We will fix this asap, till then I recommend creating the user with an @ in the username

**Comment by mzch** [comment]
Oh, when using mail address format as username, I can create a new user. Thanks! ;)
